Kinvey/Kinvey/Client.swift

```diff
@@ -266,10 +267,62 @@ open class Client: Credential {
         return Client.fileURL(appKey: self.appKey!, tag: tag)
     }
     
-    public func encode(with aCoder: NSCoder) {
```

Comment:
this is a legacy code when we had some support for Obj-C, so no need to keep it.